packages/styles/scss/patterns/sub-patterns/card/index.scss

```diff
@@ -27,16 +27,29 @@
     }
 
     &__title {
+      margin-bottom: $carbon--layout-05;
       @include carbon--type-style('expressive-heading-03');
+    }
 
-      margin-bottom: $carbon--layout-05;
+    &__title,
+    &__content {
+      color: $text-01;
     }
 
     &__content {
+      margin-bottom: $carbon--layout-01;
       @include carbon--type-style('body-short-02');
+    }
 
-      margin-bottom: $carbon--layout-01;
+    &__eyebrow {
       color: $text-02;
+      margin-bottom: $carbon--spacing-03;
```

Comment:
Should we be consistent with using layout vs. spacing? Change to one or the other. I feel like spacing might be more appropriate here. Can't remember why I chose layout originally.

Reply:
hmm looking at carbon's website, it seems to me that `spacing` is used more for within a component and `layout` is for between components

https://www.carbondesignsystem.com/guidelines/spacing#spacing-scale

I'll change the rest of the tokens to use `spacing` instead